cerberus/cerberus.py

```diff
@@ -208,6 +208,37 @@ def _validate(self, document, schema=None, update=False, context=None):
         elif self.schema is None:
             raise SchemaError(errors.ERROR_SCHEMA_MISSING)
 
+
+        # if self.schema is a list, then we want to validate the document against each item
+        # in the list and return true if one schema in the list validates.
+        if isinstance( self.schema, Sequence ) and not isinstance(self.schema, _str_type):
+            # assume that we no schemas validate
+            valid = False
+            # keep an error stack to pass on in case no schemas validate
+            errorstack = []
+            # for each schema...
+            for s in self.schema:
+                validator = copy.copy(self)
```

Comment:
Got it, I copied the loop from `_validate_schema` function. If `value` is a `Mapping`, the `copy.copy(self)` is used. Perhaps that should be changed to?